src/pages/DispensingPage.js

```diff
@@ -119,13 +131,25 @@ const Dispensing = ({
           getItemLayout={getItemLayout}
         />
       </DataTablePageView>
-      <ModalContainer title="Patient Details" noCancel fullScreen isVisible={modalOpen}>
+      <ModalContainer title="Patient Details" noCancel fullScreen isVisible={patientModalOpen}>
         <FormControl
           onSave={isCreating ? saveNewPatient : savePatient}
           onCancel={cancelPatientEdit}
           inputConfig={getFormInputConfig('patient', currentPatient)}
         />
       </ModalContainer>
+      <ModalContainer
+        title="Prescriber Details"
+        noCancel
```

Comment:
hm? there's a `onCancel` method supplied below

Reply:
that's confusing.. the `noCancel` on the `ModalContainer` indicates that the `X` on the top right shouldn't display. The `onCancel` on the form control is the even t handler for the `cancel` button on the bottom right.. not sure how to make that clearer though

Reply:
so `noCancel` means `oneCancel` and both cancels use the `onCancel`. I get it now.
I think that's clear enough as it is.